lighthouse-cli/test/cli/index-test.js

```diff
@@ -82,8 +82,46 @@ describe('CLI Tests', function() {
   });
 
   describe('print-config', () => {
+    describe('fraggle rock', () => {
```

Comment:
I created a "fraggle rock" block instead of a "legacy" block so the snapshots would be preserved.